invenio_files_rest/serializer.py

```diff
@@ -268,7 +269,8 @@ def _whitespace_waiting():
 
 
 def json_serializer(data=None, code=200, headers=None, context=None,
-                    etag=None, task_result=None):
+                    etag=None, task_result=None,
+                    serializer_mapping=serializer_mapping, view_name={}):
```

Comment:
I guess that if we go for the param `view_name`, and not `kwargs`, I would put a string and not a dict...